exitstats_test.py

```diff
@@ -0,0 +1,55 @@
+"""Tests for sidestream."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import os
+import unittest
+import time
+
+import exitstats
+
+class ExitstatsTest(unittest.TestCase):
+
+  def testSetkey(self):
+    exitstats.setkey({'foo':3, 'bar':2, 'baz':1})
+    self.assertEqual(sorted(exitstats.active_vars), ['bar', 'baz', 'foo'])
+    # stdvars should appear before others...
+    exitstats.setkey({'MinA':3, 'MinB':2, 'MinRTT':1})
+    self.assertEqual(exitstats.active_vars[0], 'MinRTT')
+    self.assertTrue('MinA' in  exitstats.active_vars)
+
+  def testGetlogf(self):
```

Comment:
Would then `freezegun` library be useful here? e.g. https://github.com/m-lab/scraper/blob/master/scraper_test.py#L190